[WIP] Add DUO credential

aykevl opened the pull request:
Initial DUO credential, for review. Untested. It is roughly a mix of iDIN and Surfnet, with some extra attributes added.

Questions:

* I couldn't find an 'official' translation of "Dienst Uitvoering Onderwijs" so I'm not sure what English name I should give it.
* I'm not entirely sure the `archieved` attribute has the right ID. It's the month this degree has been archieved. Also, what's the syntax for a month? I currently use `mm-yyyy` but I really think dates should be in RFC3339/ISO format.
* `institute` looks like it's actually `<institute> in <city>`, maybe we should split that out. I need more example documents for that to be certain.
* There are two attributes I haven't yet included, which I'm not entirely sure how to do. These are "Soort waardedocument" ("Getuigschrift") and "Aard van het examen" ("WO Master"). Again, I'll need more examples to know the possible values.
* I couldn't easily find a good icon, only a logo.

sietseringers commented:
* The credential is now called 'Dienst Uitvoering Onderwijs'/'DUO', i.e. it is named after the party from which the data comes, not after what it contains. Wouldn't 'University diploma'/'diploma' be better? (Or is 'degree' a more suitable word for this in English?)
* Achieving is spelled without an r (i.e. not "archieving").

aykevl commented:
Modified the PR. Applied your suggestions, and added the following attributes:

* prefix (tusssenvoegsel)
* degree (havo, vwo, WO bachelor, WO master, etc.)
* profile (middelbare school profiel)
* city (where the institute is located, in all-caps)

aykevl commented:
Updated again, removing the profile attribute. The 'havo, vwo' part of the degree is part of the education attribute and I've put the profile information in the degree attribute.
I think this is roughly what we agreed on, let me know if you think we should do this differently.
